### PR TITLE
feat: isEmpty for strings

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: src/main/java/nva/commons/utils/StringUtils.java [hunk 1]
```diff
@@ -35,4 +37,8 @@ public static String removeMultipleWhiteSpaces(String input) {
     public static String removeWhiteSpaces(String input) {
         return input.replaceAll(WHITESPACES, EMPTY_STRING);
     }
+
```
Comment (brinxmat) [inline src/main/java/nva/commons/utils/StringUtils.java:40]: \s matches \t \n \r, there are cases where we want medial whitespaces, but not these or leading/trailing spaces…I implemented this in a recent PR, but I guess it belongs here

File: src/test/java/nva/commons/utils/StringUtilsTest.java [hunk 2]
```diff
@@ -46,4 +46,35 @@ public void removeWhiteSpacesRemovesWhiteSpaces() {
         String output = StringUtils.removeWhiteSpaces(combinationOfSpaces);
         assertThat(output, is(equalTo(expected)));
     }
+
```
Comment (brinxmat) [inline src/test/java/nva/commons/utils/StringUtilsTest.java:49]: These could be ParametrizedTest and NullAndEmptySource

File: src/main/java/nva/commons/utils/StringUtils.java (hunk 1 above)
Comment (axthosarouris) [reply src/main/java/nva/commons/utils/StringUtils.java:40]: We can fix this, in another PR.

File: build.gradle [hunk 3]
```diff
@@ -64,14 +64,14 @@ dependencies {
     implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.9'
 
     testImplementation group: 'com.github.BIBSYSDEV', name: 'nva-testutils', version: '0.1.4'
-    testRuntimeOnly(group: 'org.junit.jupiter', name: 'junit-jupiter', version: jupiterVersion) {
+
+    testImplementation(group: 'org.junit.jupiter', name: 'junit-jupiter', version: jupiterVersion) {
```
Comment (axthosarouris) [inline build.gradle:68]: testRuntimeOnly is not sufficient, gradle complains that it does not find the annotations. Strangely enough gradle lint complains that the whole library is not necessary

**Pull request merged**